tests/js/utils.js

```diff
@@ -0,0 +1,237 @@
+
+import {Strip, strip, Cookies, cookies, DjangoAjax, PontoonDjangoAjax, ajax} from 'utils';
+
+
+test('PontoonDjangoAjax instance', () => {
+    expect(ajax instanceof DjangoAjax).toBe(true);
+    expect(ajax instanceof PontoonDjangoAjax).toBe(true);
+})
+
+
+test('DjangoAjax csrf', () => {
+    const _ajax = new DjangoAjax();
+    document.cookie = "csrf=tripe";
+    expect(_ajax.csrf).toBe('tripe');
+})
+
+
+test('PontoonDjangoAjax csrf', () => {
+    const _ajax = new PontoonDjangoAjax();
+    document.querySelector = jest.fn(() => {return {value: 'rhubarb'}})
```

Comment:
do arrow functions require return? I think that:
`jest.fn(() => ({value: 'rhubarb'}))`
could be shorter.

Reply:
how so ?

Reply:
ah - i see the question properly - yep if you have `{}` around the value

Reply:
the alternate being  ` jest.fn(() => {return {value: 'rhubarb'}})`

Reply:
wow - i think i read this late at night - apologies - i see you suggested the shorter version - updating...